client/src/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import '../styles/style.scss';
+import { Instagram } from 'react-content-loader';
+
+export interface IUserData {
+    first_name: string;
+    last_name: string;
+    avatar: string;
+}
+
+interface IFormProps {
+    getUser: () => void;
+    user: IUserData;
+    loading: boolean;
+    loaded: boolean;
+}
+
+export default class Profile extends React.Component<IFormProps> {
+
+    public state: {loading: boolean} = {
+        loading: this.props.loading,
+    };
+    public timerHandle: any = 0;
+
+    public componentDidMount(): void {
+        this.props.getUser();
+
+    }
+    public componentDidUpdate(): void {
+        if (this.props.loading && this.props.loaded) {
+            this.timerHandle = setTimeout(() => {
+                this.setState({loading: false});
+                this.timerHandle = 0;
+            },
+                3000,
+            );
+        }
+    }
+    public componentWillUnmount(): void {
+        clearTimeout(this.timerHandle);
+        this.timerHandle = 0;
+    }
+
+    public render(): JSX.Element {
+        const { first_name, last_name, avatar }: any = this.props.user;
+        const { loading }: {loading: boolean} = this.state;
```

Comment:
you can not write type after constant declaration

Reply:
tslint expected type after destructuring